components/docs-chef-io/layouts/shortcodes/automate/automate_cli_commands_ha.html

```diff
@@ -0,0 +1,104 @@
+{{ $data_path := index $.Site.Data (split (.Get "data_path") "/") }}
```

Comment:
You shouldn't need to duplicate this. You can just point the automate_cli_commands shortcode at a different data set.